app/assets/stylesheets/typeahead.scss

```diff
@@ -1,6 +1,6 @@
 .tt-menu {
   width: 300px;
-  margin-top: 11px;
+  margin-top: ($navbar-height - $input-height-base + 4) / 2;
```

Comment:
`margin-top: ($navbar-height - $input-height-small) / 2;` would make more sense to me. The value is the same but it is not obvious why your code does the right thing and it will break with a higher probability whenever the Bootstrap variables change.